w3af/plugins/attack/db/sqlmap/lib/core/wordlist.py

```diff
@@ -47,7 +47,7 @@ def adjust(self):
                     errMsg = "something appears to be wrong with "
                     errMsg += "the file '%s' ('%s'). Please make " % (self.current, getSafeExString(ex))
                     errMsg += "sure that you haven't made any changes to it"
-                    raise SqlmapInstallationException, errMsg
+                    raise SqlmapInstallationException as errMsg
```

Comment:
Bug

Reply:
Also, please do not change anything inside the `w3af/plugins/attack/db/sqlmap` directory. That is sqlmap code which is inside w3af's repo. If we change it, next time I merge the latest sqlmap into w3af's repo the merge will conflict